### PR TITLE
[CloudSAASOpenbeta] - Change user menu organization navigation for open beta

### PR DESCRIPTION
**Pull request by pietrorosa77**
This PR it's the first of a series of pr for the Open beta release. We are changing the user navigation menu and account menu removing the Organization list and adding a "your organizations" entry that will redirect the user to a page with the list of organizations the user belongs to

https://sourcegraph.atlassian.net/browse/CLOUD-296
https://sourcegraph.atlassian.net/browse/CLOUD-297

image

## Test plan

The code is entirely behind a Feature flag and it has no impact on the existing

### CODE REVIEW

File: client/web/src/org/members/OrgMembersListPage.tsx [hunk 1]
```diff
@@ -255,7 +256,7 @@ export const OrgMembersListPage: React.FunctionComponent<Props> = ({ org, authen
                     )}
                 </div>
 
-                <Container className={styles.membersList}>
+                <Container className={classNames(!showOnlyYou ? 'mb-3' : undefined, styles.membersList)}>
```
Comment (kopancek) [inline client/web/src/org/members/OrgMembersListPage.tsx:259]: This is shorter and more readable IMO
```suggestion
                <Container className={classNames({ 'mb-3' : !showOnlyYou}, styles.membersList)}>
```

File: client/web/src/org/members/OrgMembersListPage.tsx [hunk 2]
```diff
@@ -282,36 +283,33 @@ export const OrgMembersListPage: React.FunctionComponent<Props> = ({ org, authen
...
-                                        orgId={org.id}
-                                        onInviteSent={onInviteSent}
-                                        className={styles.inviteMemberLink}
-                                        as="a"
-                                        size="lg"
-                                        variant="link"
-                                    />
-                                    {` to join you on ${org.name} on Sourcegraph`}
-                                </div>
+                {authenticatedUser && membersResult && showOnlyYou && isSelf(membersResult.members.nodes[0].id) && (
```
Comment (kopancek) [inline client/web/src/org/members/OrgMembersListPage.tsx:293]: Why do we need to check if the member `isSelf`? Theoretically only the org member can see this page, right?

**Review by kopancek — APPROVED** [review]
Looks good to me, 2 non-blocking comments

File: client/web/src/org/members/OrgMembersListPage.tsx (hunk 2 above)
Comment (pietrorosa77) [reply client/web/src/org/members/OrgMembersListPage.tsx:293]: in this case the banner is going to be shown only if we have 1 user and that 1 user is yourself. the banner says "looks like there's only you here" so if there's only 1 member it needs to be you

File: client/web/src/org/members/OrgMembersListPage.tsx (hunk 1 above)
Comment (pietrorosa77) [reply client/web/src/org/members/OrgMembersListPage.tsx:259]: it's much better so i'm going to add it

**Pull request merged**